website/content/docs/k8s/deployment-configurations/argo-rollouts-configuration.mdx

```diff
@@ -0,0 +1,219 @@
+---
+layout: docs
+page_title: Using Argo Rollouts for Canary upgrades with Consul on Kubernetes
```

Comment:
```suggestion
page_title: Argo Rollouts Progressive Delivery with Consul on Kubernetes
```

This title I believe seems to align with other Argo Rollouts documentation so proposing.